client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -57,6 +57,13 @@ class JetpackChecklist extends PureComponent {
 		}
 	};
 
+	handleWpAdminLink = () => {
```

Comment:
I'd keep this in the other component. It's probably _fine_ to keep it as a function component with this defined in the body, or it can become a `PureComponent` that keeps this method.

Reply:
Wouldn't I then need to `connect` for `recordTracksEvent` and we'd be back to https://github.com/Automattic/wp-calypso/pull/32874/commits/3dfb786e469668a8706f4896e1e011f884d0cc64 ?

Reply:
Oh, right... Ignore me.